testing/time.ts

```diff
@@ -0,0 +1,436 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+/** This module is browser compatible. */
+
+import { ascend, RBTree } from "../collections/rb_tree.ts";
+import { DelayOptions } from "../async/delay.ts";
+
+export const _internals = {
```

Comment:
Why do these need to be exported?

Reply:
They are used by the test coverage to verify the real methods are called or not called when they should be. I believe the tests might replace them with stubs so that the real timer functions are never called. I could move them to a separate file with _ prefix to the name so that this constant is never exposed to end users.

Reply:
I see, that makes sense, though I can see this not working properly if user's code replaced them before this file is loaded. I think this is fine as is, can you add a comment describing the purpose?

Reply:
Sure I can do that tonight and I think I'll just move them to _test_utils.ts. alternatively I could create a new file called _time.ts that could export just it instead.

Reply:
`_test_utils.ts` seems fine 👍

Reply:
I thought about this and think _time.ts would be better so that when people import time.ts it won't have to load _test_utils.ts too which contains irrelevant utilities and could potentially grow in the future.

Reply:
I moved the _internals const to _time.ts and added a comment describing its purpose.